### PR TITLE
[IMP] new filter on attributes by 'unknown' attribute value, is considered 'unknown' a product that has an attribute but has no value on it. HU#1006 CA#41

### PR DESCRIPTION
**Pull request by oscarolar**
FIXES https://github.com/vauxoo/yoytec/issues/1099
BUILD https://github.com/Vauxoo/yoytec/pull/1222

### CODE REVIEW

**Comment by moylop260** [comment]
Could you add a message of why the coverage is down in `website_product_filters/models/website.py` from dummy?

Show `addons-vauxoo/website_product_filters/models/website.py` 41% of coverage and the missed lines are: 45-51, 53-63

File: website_product_filters/models/website.py [hunk 1]
```diff
@@ -48,5 +49,17 @@ def sale_product_domain(self):
...
+            unknown_list = request.httprequest.args.getlist('unknown')
+            values = [map(int, v.split("-")) for v in unknown_list if v]
+            if values:
+                ids = []
+                for value in values:
+                    if value[0] not in ids:
+                        ids.append(value[0])
+            line_ids = line_obj.search([('attribute_id', 'in', ids),
+                                        ('value_ids', '=', False)])
+            unknown_domain += [('attribute_line_ids', 'in', line_ids._ids)]
```
Comment (moylop260) [inline website_product_filters/models/website.py]: Use `list.append(tuple)` is faster than `list += list`

**Comment by oscarolar** [comment]
Coverage decreased on this PR wil be reincreased when this issue gets fixed https://github.com/Vauxoo/yoytec/issues/1162 please merge as it is now.

**Comment by moylop260** [comment]
Ok, please create the new issue to fix the coverage

**Pull request merged**